test/integration/protocol_integration_test.cc

```diff
@@ -1600,7 +1600,6 @@ TEST_P(DownstreamProtocolIntegrationTest, ManyRequestHeadersAccepted) {
 TEST_P(DownstreamProtocolIntegrationTest, ManyRequestTrailersRejected) {
   // QUICHE doesn't limit number of headers.
   EXCLUDE_DOWNSTREAM_HTTP3
```

Comment:
The upstream http3 test failure is irrelevant to this fix right? I doubt current fix works for trailers.